flink-metrics/flink-metrics-prometheus/src/main/java/org/apache/flink/metrics/prometheus/PrometheusPushGatewayReporterOptions.java

```diff
@@ -54,7 +54,14 @@
             ConfigOptions.key("deleteOnShutdown")
                     .defaultValue(true)
                     .withDescription(
-                            "Specifies whether to delete metrics from the PushGateway on shutdown.");
+                            Description.builder()
+                                    .text(
+                                            "Specifies whether to delete metrics from the PushGateway on shutdown."
+                                                    + " Flink will try its best to delete the metrics but this is not guaranteed. See here %s",
+                                            LinkElement.link(
+                                                    "https://issues.apache.org/jira/browse/FLINK-13787",
+                                                    "for more details."))
```

Comment:
The link should be attached to "here", not to "for more details".

Reply:
got it, thx